pkg/ipam/ippool.go

```diff
@@ -26,14 +26,15 @@ import (
 	"sync"
```

Comment:
This file is being moved to admiral : https://github.com/submariner-io/admiral/pull/982. I can make these changes with my PR or you can submit a new one.